rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -26,16 +26,19 @@
 
 
 class PyPIURLFixHook(BaseSpecHook):
-    """
-    SpecHook for transforming old nonfunctional python package url to
-    a safe url(eg. https://pypi.* to https://files.pythonhosted.*)
+    """SpecHook for transforming an old python package url to a new url(eg. https://pypi.* to
+    https://files.pythonhosted.* or https://pypi.python.org/pypi/* to https://pypi.org/project/*)
+
     """
 
     NAME = 'PyPI URL Fix'
     CATEGORIES = ['python']
     SOURCES_URL_TRANSFORMATIONS = [
         (r'https?://pypi(\.python)?\.org/', 'https://files.pythonhosted.org/'),
     ]
+    URL_TRANSFORMATIONS = [
+        (r'https?://pypi(\.python)?\.org/pypi/', 'https://pypi.org/project/'),
```

Comment:
I don't think anybody is using `https://pypi.org/pypi`, as that never has been a valid URL AFAIK.